web/src/components/HomeSidebar/TagsSection.tsx

```diff
@@ -24,6 +24,22 @@ const TagsSection = () => {
     [memoList.size()],
   );
 
+  const [search, setSearch] = useState<string | null>();
+
+  const handleDeleteTag = async (tagValue: string) => {
```

Comment:
You can wrap this function in the [useCallback](https://react.dev/reference/react/useCallback) to cache it between re-renders